allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_ophys_data_transforms.py

```diff
@@ -192,28 +188,33 @@ def get_metadata(self) -> Dict[str, Any]:
         """Return metadata about the session.
         :rtype: dict
         """
+        extractor = self.extractor
+
         behavior_session_uuid = self.get_behavior_session_uuid()
-        fov_shape = self.extractor.get_field_of_view_shape()
-        expt_container_id = self.extractor.get_experiment_container_id()
+        fov_shape = extractor.get_field_of_view_shape()
+        expt_container_id = extractor.get_experiment_container_id()
 
         metadata = {
-            'ophys_experiment_id': self.extractor.get_ophys_experiment_id(),
+            'behavior_session_id': extractor.get_behavior_session_id(),
```

Comment:
If you have time, it might be useful here to do something along the lines of:

```
behavior_metadata = super().get_metadata()

# ophys-specific metadata
ophys_metadata = {
   ...
}

return {**behavior_metadata, **ophys_metadata}
```

in order to reduce duplication.